interactive/views.py

```diff
@@ -191,7 +194,66 @@ def get_object(self, queryset=None):
 
     def get_context_data(self, **kwargs):
         report = process_html(self.object.report_content)
+        update_url = reverse(
+            "interactive:report-update",
+            kwargs=dict(
+                slug=self.object.slug,
+                project_slug=self.object.project.slug,
+                org_slug=self.object.project.org.slug,
+            ),
+        )
```

Comment:
I think for OSI we can put this on the Report model as `get_edit_url` if we have a note that it might need changing for more generic reports?

Reply:
So, this was the crux of my uncertainty - is editing an OSI report the same thing as a general Report?

If it's just title/description, then I think yes? As that's exactly the kind of metadata we have on the current reports site.

But if its more, we might to specialise urls/views for OSI report editing?

Should we assume the same for now? In which case, should we have this view in jobserver/views/reports.py, and the url based on the report?

Reply:
I think that's definitely a reasonable concern to have!

I suspect we might revert this once reports in general are all in job-server, but for now I was just thinking this would be a convenience wrapper like our other `get_FOO_url` ones.  Maybe that's a little confusing though 🤔  What about putting a comment above this to point out this exact discussion, just something short like:

> TODO: decide if this should be the canonical way to edit reports once reports arrive in this project

?